test/e2e/framework/test_context.go

```diff
@@ -41,9 +41,9 @@ func registerFlags(t *TestContextType) {
 	flag.StringVar(&TestContext.ReportPrefix, "report-prefix", "", "Optional prefix for JUnit XML reports. Default is empty, which doesn't prepend anything to the default name.")
 	flag.StringVar(&TestContext.ReportDir, "report-dir", "", "Path to the directory where the JUnit XML reports should be saved. Default is empty, which doesn't generate these reports.")
 	flag.StringVar(&TestContext.SubmarinerNamespace, "submariner-namespace", "submariner", "Namespace in which the submariner components are deployed.")
-	flag.UintVar(&TestContext.ConnectionTimeout, "connection-timeout", 6, "The timeout in seconds per connection attempt when verifying communication between clusters.")
-	flag.UintVar(&TestContext.ConnectionAttempts, "connection-attempts", 10, "The number of connection attempts when verifying communication between clusters.")
-	flag.UintVar(&TestContext.OperationTimeout, "operation-timeout", 60, "The general operation timeout in seconds.")
+	flag.UintVar(&TestContext.ConnectionTimeout, "connection-timeout", 60, "The timeout in seconds per connection attempt when verifying communication between clusters.")
```

Comment:
As tom said, in case of failure this will be long 60*4 = 2 minutes.

But it should be fine on ok-cases, I'm trying to avoid false negatives happening sometimes because
the listener timeouts while the connector is being spawned, 6 seconds is too short.

Reply:
It was 6 sec but with 10 attempts for total 1 min.  I'm fine with increasing too 2 min.  I'm just not sure if it's better to have shorter timeout and more attempts or vice versa or if it really matters. Let's see how this works.